src/plugins/settings.rs

```diff
@@ -178,6 +180,31 @@ impl SettingsPlugin {
                         });
                     });
                 });
+                if data.dev_mode {
+                    frame(ui, |ui| {
+                        ui.columns(2, |ui| {
+                            "disable console output".to_colored().label(&mut ui[0]);
```

Comment:
Since buttons say "enabled"/"disabled" option should be called just "console output"